packages/clerk-js/src/core/auth/CaptchaHeartbeat.ts

```diff
@@ -0,0 +1,45 @@
+import { createWorkerTimers } from '@clerk/shared/workerTimers';
+
+import { CaptchaChallenge } from '../../utils/captcha/CaptchaChallenge';
+import type { Clerk } from '../resources/internal';
+
+export class CaptchaHeartbeat {
+  constructor(
+    private clerk: Clerk,
+    private captchaChallenge = new CaptchaChallenge(clerk),
+    private timers = createWorkerTimers(),
+  ) {}
+
+  public async start() {
+    if (!this.isEnabled()) {
+      return;
+    }
+
+    await this.sendCaptchaToken();
+    this.timers.setInterval(() => {
+      void this.sendCaptchaToken();
+    }, this.intervalInMs());
+  }
+
+  private async sendCaptchaToken() {
```

Comment:
@nikosdouvlis Can we rename this function to `triggerInvisibleCaptcha` or some sort, because it's a bit confusing only to mention `send`.